.github/workflows/ci.yml

```diff
@@ -81,7 +67,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '2.7'
+          ruby-version: '3.4'
```

Comment:
I'd rather keep these on the lower supported versions I think. Is there a reason you want to change it?

Reply:
Just thought I'd use the latest ruby, happy to revert.